### PR TITLE
WIP: Have the ability to print JSON, but need to test ability to prin…

### PR DESCRIPTION
**Pull request by ericdrosado**
…t a server directory without using a real server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
-# ElixirClient
+# Elixir Client
 
-**TODO: Add description**
+## Project Requirements
++ Swift 1.6.4+
```
Comment (cdemyanovich) [inline README.md:4]: Should "Swift" be "Elixir"?

**Review by cdemyanovich — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+### Build Project
++ In the root directory `mix escript.build`.
 
-Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
-and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
-be found at [https://hexdocs.pm/elixir_client](https://hexdocs.pm/elixir_client).
+## Run Client
 
+### Run
++ In the root directory: `./elixir_client --host=https://erosado-swift-server.herokuapp.com`
```
Comment (cdemyanovich) [inline README.md:17]: I got this error when running:
```
➜  Elixir-Client git:(communicate_with_server) ✗ ./elixir_client --host=https://erosado-swift-server.herokuapp.com
** (KeyError) key :body not found in: %HTTPoison.Error{id: nil, reason: :timeout}
    (elixir_client) lib/client/client.ex:5: Client.get/1
    (elixir) lib/kernel/cli.ex:105: anonymous fn/3 in Kernel.CLI.exec_fun/2
```

File: README.md [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
 
-```elixir
-def deps do
-  [
-    {:elixir_client, "~> 0.1.0"}
-  ]
-end
-```
+### Build Project
++ In the root directory `mix escript.build`.
```
Comment (cdemyanovich) [inline README.md:12]: Why are there changes after I compile?
```
➜  Elixir-Client git:(communicate_with_server) ✗ git st
On branch communicate_with_server
Your branch is up to date with 'origin/communicate_with_server'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   elixir_client

no changes added to commit (use "git add" and/or "git commit -a")
➜  Elixir-Client git:(communicate_with_server) ✗
```

**Review by tommcgee — CHANGES_REQUESTED** [review]
Can't get this out of my pending PR reviews! Hope all is well!

**Comment by ericdrosado** [comment]
> Can't get this out of my pending PR reviews! Hope all is well!

Whoops! Sorry about that @tommcgee !

I hope all is well.

**Pull request closed without merging**